cmd/parser/fake_yml.go

```diff
@@ -0,0 +1,72 @@
+package parser
```

Comment:
haven't we started adding `build ignore` tags to fake files?

Reply:
we have a separated issue for that #224